src/browser/settings.jsx

```diff
@@ -56,6 +71,14 @@ var SettingsPage = React.createClass({
       var currentWindow = remote.getCurrentWindow();
       currentWindow.setAutoHideMenuBar(config.hideMenuBar);
       currentWindow.setMenuBarVisibility(!config.hideMenuBar);
+
+      var autostart = this.state.autostart;
+      appLauncher.isEnabled().then(function(enabled) {
```

Comment:
Don't think so, if would be very confusing if it gets set of you don't click the option.